packages/xo-web/src/common/xo/index.js

```diff
@@ -3303,6 +3304,30 @@ export const deployProxyAppliance = (license, sr, { network, proxy, ...props } =
     ...props,
   })::tap(subscribeProxies.forceRefresh)
 
+export const registerProxy = async () => {
+  const registerProxyInfo = await confirm({
+    body: <RegisterProxyModal />,
+    icon: 'connect',
+    title: _('registerProxy'),
+  })
+
+  const proxyId = await registerProxyApplicance(registerProxyInfo)
+  const _isProxyWorking = await isProxyWorking(proxyId).catch(err => {
+    console.error(err)
```

Comment:
Good idea :+1: 

But we can make it easier to spot in the logs:

```suggestion
    console.error('isProxyWorking error:', err)
```